src/components/UI/FormSelectorFilterable.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <label v-if="label" :for="id" :class="$style.label">
+      {{ label }}
+    </label>
+    <!-- https://vue-select.org/api/props.html -->
+    <v-select
+      :id="id"
+      v-model="selectedOptionValue"
+      :options="options"
+      label="key"
+      :reduce="(option: Option) => option.value"
+      :selectable="(option: Option) => option.value !== null"
+      :clearable="false"
+      :data-background="background"
+      :class="$style.select"
+    ></v-select>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { useModelValueSyncer } from '/@/composables/useModelSyncer'
+import { randomString } from '/@/lib/basic/randomString'
+import vSelect from 'vue-select'
+import 'vue-select/dist/vue-select.css'
+
+type Option = { key: string; value: string | null }
+type Background = 'primary' | 'secondary'
+const props = withDefaults(
+  defineProps<{
+    modelValue?: string | null
+    background?: Background
+    options: Array<Option>
+    label?: string
+  }>(),
+  {
+    modelValue: '',
+    background: 'secondary'
+  }
+)
+
+const emit = defineEmits<{
+  (e: 'update:modelValue', _val: string | null): void
+}>()
+
+const selectedOptionValue = useModelValueSyncer(props, emit)
+const id = randomString()
+</script>
+
+<style lang="scss" module>
+.label {
+  @include color-ui-primary;
+  margin-bottom: 4px;
+  display: block;
+}
+// https://vue-select.org/guide/css.html
+.select {
+  // background
+  @include background-secondary;
+  &[background='primary'] {
```

Comment:
CSSの方も全部`data-background`に直してほしいです
```suggestion
  &[data-background='primary'] {
```

Reply:
ごめんなさい忘れてました、直します

Reply:
propsの`background`って命名このままでいいですか？

Reply:
propsの方はこのままでOKです👍

Reply:
了解です！